exchanges/liqui/liqui.go

```diff
@@ -34,7 +34,7 @@ const (
 	liquiWithdrawCoin      = "WithdrawCoin"
 
 	liquiAuthRate   = 0
-	liquiUnauthRate = 0
+	liquiUnauthRate = 1
```

Comment:
Is liqui prone to authentication errors?

Reply:
Unauthenticated rate limits were being exceeded, I just bumped it up to one. When we have the trading engine online and processing we can implement an auto adjustment function for the rates.

Reply:
Thanks for the clarification!